packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -371,26 +371,23 @@ function addEditableCallbacks<T extends DbObject>({collection, options = {}}: {
       const userId = currentUser._id
       const editedAt = new Date()
       const changeMetrics = htmlToChangeMetrics("", html);
-      
-      // FIXME: This doesn't define documentId, because it's filled in in the
-      // after-create callback, passing through an intermediate state where it's
-      // undefined; and it's missing _id and schemaVersion, which leads to having
-      // ObjectID types in the database which can cause problems. Would be good
-      // to remove this ts-ignore, since there was recently an important bug here
-      // (missing fieldName) that typechecking would have caught.
-      // @ts-ignore
-      const firstRevision = await Connectors.create(Revisions, {
-        ...await buildRevision({
+      const newRevision: Omit<DbRevision, "documentId" | "schemaVersion" | "_id"> = {
+        ...(await buildRevision({
           originalContents: doc[fieldName].originalContents,
           currentUser,
-        }),
+        })),
         fieldName,
         collectionName,
         version,
-        updateType: 'initial',
+        updateType: "initial",
```

Comment:
Sorry, this was probably my autoformatter again. Not sure if it's worth reverting to reduce diff noise.